sklearn/plot/tests/test_heatmap.py

```diff
@@ -0,0 +1,24 @@
+from sklearn.plot import plot_heatmap
+from sklearn.utils.testing import SkipTest
+import numpy as np
+
+
+def test_heatmap():
+    try:
+        import matplotlib
+    except ImportError:
+        raise SkipTest("Not testing plot_heatmap, matplotlib not installed.")
+
+    import matplotlib.pyplot as plt
```

Comment:
hm it looks like matplotlib is not installed for the service that computes coverage? hm...